wolvic/browser/tab_jni.cc

```diff
@@ -34,4 +38,18 @@ base::android::ScopedJavaLocalRef<jobject> JNI_Tab_CreateWebContents(
   return web_contents.release()->GetJavaWebContents();
 }
 
+void JNI_Tab_SetWebContentsDelegate(
+    JNIEnv* env,
+    const JavaParamRef<jobject>& jweb_contents,
+    const JavaParamRef<jobject>& jweb_contents_delegate) {
+  WebContents* web_contents = WebContents::FromJavaWebContents(jweb_contents);
+  static std::unique_ptr<WebContentsDelegateAndroid> web_contents_delegate;
```

Comment:
Why is this static?

Reply:
Ah. This is to temporarily keep WebContentsDelegateAndroid lifecycle because the tab implementation has not yet been reflected. Once implemented, this variable should be moved to the member in the Tab class.